### PR TITLE
changed string to int matrix size input

### PR DESCRIPTION
**Pull request by ZachLamb**
@calebhsu , @seanhansberry , @IanChar 

I used "parsedInt" to change what I'm pushing to Firebase

### CODE REVIEW

**Review by calebhsu — DISMISSED** [review]
I'm getting Permission Denied when resizing numCols.
Also, extra console.log() on line 22 (idk if that's the issue)

@calebhsu , the issue is that you haven't selected a grid at this point I believe. Should I grey out the textfields until the user clicks on a grid?

@ZachLamb yeah grey out or even hide it since it's not even necessary. we'll improve the splash page at some point. thanks!

File: src/components/matrixSize.jsx [hunk 1]
```diff
@@ -17,9 +17,9 @@ export default class MatrixSize extends React.Component {
       this.setState({ numCols: event.target.value });
   }
   handleClick(){
-  firebase.database().ref('grids/' + this.state.gridId + '/numRows' ).set(this.state.numRows);
-  firebase.database().ref('grids/' + this.state.gridId + '/numCols' ).set(this.state.numCols);
-
+  firebase.database().ref('grids/' + this.state.gridId + '/numRows' ).set(parseInt(this.state.numRows));
+  firebase.database().ref('grids/' + this.state.gridId + '/numCols' ).set(parseInt(this.state.numCols));
+  console.log()
```
Comment (sperry94) [inline src/components/matrixSize.jsx]: remove this line

**Review by sperry94 — DISMISSED** [review]
remove console log. also, the state objects for numRows/numCols all need to be strings otherwise it will throw some errors

File: src/components/matrixSize.jsx (hunk 1 above)
Comment (ZachLamb) [reply src/components/matrixSize.jsx]: That error is coming from "Matrix" not "MatrixSize" ,but I could fix that in a different commit

**Comment by calebhsu** [comment]
@ZachLamb 
No, I definitely have grid1 selected because it says (correct branch & authenticated and everything)
"set at /grids/grid1/numRows failed: Permission denied"

@sperry94 do you get a Permission Denied as well when trying to update grid size?

File: src/components/matrixSize.jsx (hunk 1 above)
Comment (sperry94) [reply src/components/matrixSize.jsx]: I would fix that real quick since it's a related issue. It's really stupid state objects can only be strings

**Comment by ZachLamb** [comment]
@calebhsu , can you approve this?

**Pull request closed without merging**